dashboard/templatetags/cards.py

```diff
@@ -9,24 +9,34 @@
 
 from core import models
 
-
 register = template.Library()
 
 
-@register.inclusion_tag('cards/diaperchange_last.html')
-def card_diaperchange_last(child):
+def _user_wants_hide(context):
```

Comment:
Kind of nitpicky but I don't like the `user_wants_hide` naming here. Can you refactor this and the related variables as just `hide_empty`?

Reply:
I've renamed it.